### ISSUE TITLE
how i can make it the default file explorer?

### ISSUE THREAD

**Issue by mchridz** [DearVa/ExplorerEx#41]
hello how i can make it the default file explorer? also can you remove that emoji from icon?

**Comment by DearVa** [comment]
It's **NOT** recommanded to set ExplorerEx as default because it's in develop and not stable yet.
If you really want to, this can be done by modifing the registry. you can **EDIT** `ExplorerEx/ExplorerEx/Reg/SetEx.reg` and **REPLACE** `G:\\Source\\C#\\ExplorerEx\\ExplorerEx\\build\\ExplorerEx.exe` to the actual ExplorerEx excutable path in your computer. **REMIND** that use \\\\ as the path separator.
You can restore it by `ExplorerEx/ExplorerEx/Reg/Restore.reg`. However, I only tested it on Windows 11. It's recommanded to backup the RegKeys in `SetEx.reg` and make your own `Restore.reg`.
After changing the registry, pressing `win + e` or opening explorer from start menu should launch ExplorerEx. However, when some application launch explorer directly (for example, the folder will open after unzipping an achieve), ExplorerEx will not launch. I may fix this by developing a Shell Extension later.

As for the Icon, I'm not a professional icon designer so I'm not going to change it by myself. Any idea of a new icon is appreciated!

Best wishes : )

**Comment by DearVa** [comment]
Now you can set it to default by switching on in the settings!

**Comment by mchridz** [comment]
> Now you can set it to default by switching on in the settings!

yes i can see  (BHO plugin ExplorerEx new version 0.4) but it's not working, delete explorer proxy.reg file is created. but still not replaced by default file explorer.
can you tell me how i can do manually in regedit thanks

**Comment by DearVa** [comment]
After switch on set default, a `.reg` file should be created in the ExplorerEx executable folder, you can use this to manually set default.

**Comment by mchridz** [comment]
thanks it's working now, but there's a little bug when default file explorer appear in 1 sec and disappear.
>see this gif screen recording 
[https://files.catbox.moe/l3y4vj.gif](url)
